.github/workflows/wheel_tests_and_release.yml

```diff
@@ -115,9 +119,19 @@ jobs:
       - uses: actions/setup-python@v5
         name: Install Python
         with:
-          python-version: "3.9"
+          python-version: "3.12"
+
+      - name: Build wheels for CPython (macOS) (x86_64)
+        if: matrix.cibw_arch == 'x86_64' && matrix.os == 'macos-latest'
+        uses: pypa/cibuildwheel@8d945475ac4b1aac4ae08b2fd27db9917158b6ce # 2.17.0
+        with:
+          output-dir: dist
+        env:
+          CIBW_BUILD: ${{ matrix.cibw_python }}-*
+          CIBW_ARCHS_MACOS: ${{ matrix.cibw_arch }}
```

Comment:
I notice that the wheels are not large in size at 4 MB currently – should we try for universal2 wheels? I guess Xcode should be able to handle that with ease, but I cannot recall issues in mind that may come with universal2 wheels.

I remember reading through threads on this that have come up in the past (xref: https://github.com/pypa/cibuildwheel/issues/705, https://github.com/pypa/cibuildwheel/issues/1333) but with NumPy 2.0 things might have changed by a lot.

Reply:
No, I don't want to spend any cycles on `universal2`, it's a pretty pointless format. The few users who may need it can use `delocate-fuse` to fuse regular wheels.